src/main/groovy/pl/allegro/tech/build/axion/release/domain/scm/ScmInitializationOptions.groovy

```diff
@@ -12,9 +12,9 @@ class ScmInitializationOptions {
 
     final String remoteUrl
 
-    private static final String ATTACH_REMOTE_PROPERTY = 'release.attachRemote'
+    static final String ATTACH_REMOTE_PROPERTY = 'release.attachRemote'
```

Comment:
It is rather `public` after change, but in addition to the fact if it is needed somewhere else with the new code `ATTACH_REMOTE_PROPERTY` is not treated as a constant anymore in Groovy. A getter will be generated.